test/integration/fixtures/blocks/core__comments__deprecated-1.json

```diff
@@ -4,8 +4,8 @@
 		"isValid": true,
 		"attributes": {
 			"tagName": "div",
-			"className": "wp-block-comments-query-loop comments-post-extra",
-			"textColor": "luminous-vivid-orange"
+			"textColor": "luminous-vivid-orange",
+			"className": "wp-block-comments-query-loop comments-post-extra"
```

Comment:
These values all match, but are reordered. I'm not sure what caused this to be different.